types_webgl_wasm.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2018 The Go Authors. All rights reserved.
```

Comment:
I think this is one of those copyrights that we can't assert (there were 2 instances)

Reply:
This file was created by:
commit c0be8194ddff434524d5d2a896990c29c92926eb
Author: Dmitri Shuralyov <dmitri@shuralyov.com>
Date:   Mon Sep 10 02:22:26 2018 -0400

Which means we should not alter the copyright to me.

Reply:
Apologies, I did not notice the file was added back then.